src/jobs/veracode_pipeline_scan.yml

```diff
@@ -7,71 +7,76 @@ parameters:
     type: string
     default: dps_alerts_security
     description: Slack channel to use for notifications.
+  docker_image_app_dir:
+    description: Directory inside the docker image where the application artifacts are saved
+    type: string
+    default: "/app"
 steps:
- - veracode_prepare_artifacts
- - run:
-     name: Download/Extract-Veracode-Pipeline-Scanner
-     command: |
-       wget https://downloads.veracode.com/securityscan/pipeline-scan-LATEST.zip -O pipeline-scan.zip
-       unzip pipeline-scan.zip -x README.md
- - run:
-     name: Download Veracode Policy
-     no_output_timeout: 1m
-     command: |
-       java -jar pipeline-scan.jar \
-       --request_policy "Ministry of Justice" \
-       --veracode_api_id "${VERACODE_API_ID}" \
-       --veracode_api_key "${VERACODE_API_KEY}"
- - run:
-     name: Scan Project
-     no_output_timeout: 30m
-     command: |
-       java -jar pipeline-scan.jar \
-       --file source.zip \
-       --policy_file Ministry_of_Justice.json \
-       --veracode_api_id "${VERACODE_API_ID}" \
-       --veracode_api_key "${VERACODE_API_KEY}" \
-       --timeout 60 \
-       --issue_details false \
-       --summary_output true
-#  Save results to circles artifact storage
-#  - run:
-#      name: Save results
-#      when: always
-#      command: |
-#        mkdir veracode
-#        mv results.json results.txt veracode
-#  - store_artifacts:
-#      path: veracode
- - slack/notify:
-     event: fail
-     channel: << parameters.slack_channel >>
-     custom: |
-       {
-         "blocks": [
-           {
-             "type": "context",
-             "elements": [
-               {
-                 "type": "mrkdwn",
-                 "text": ":circleci-${CCI_STATUS}: CircleCI job *${CIRCLE_JOB}* ${CCI_STATUS}"
-               }
-             ]
-           },
-           {
-             "type": "section",
-             "text": {
-               "type": "mrkdwn",
-               "text": "*${CIRCLE_PROJECT_REPONAME}* failed ${CIRCLE_JOB}"
-             },
-             "accessory": {
-               "type": "button",
-               "text": {
-                 "type": "plain_text",
-                 "text": "View job"
-               },
-               "url": "${CIRCLE_BUILD_URL}"
-             }
-           }
-         ]
-       }
+  - veracode_prepare_artifacts:
+      docker_image_app_dir: << parameters.docker_image_app_dir >>
+  - run:
+      name: Download/Extract-Veracode-Pipeline-Scanner
+      command: |
+        wget https://downloads.veracode.com/securityscan/pipeline-scan-LATEST.zip -O pipeline-scan.zip
+        unzip pipeline-scan.zip -x README.md
+  - run:
+      name: Download Veracode Policy
+      no_output_timeout: 1m
+      command: |
+        java -jar pipeline-scan.jar \
```

Comment:
assume java present on the node executor then

Reply:
Interesting spot - yes it has `openjdk` installed on `cimg/node:14.16-browsers`.  I'll leave as is for now as it works ok.